src/components/Exchange/OrdersList.js

```diff
@@ -229,11 +229,23 @@ export default function OrdersList(props) {
 
       const error = getOrderError(account, order, positionsMap);
       const orderId = `${order.type}-${order.index}`;
+      const orderTitle = `${order.type === INCREASE ? t`Increase` : t`Decrease`} ${indexTokenSymbol} ${
```

Comment:
Not critial
Now this place is a bit difficult to read for me, maybe put the conditions in separate variables?

Like this
```
const orderTypeText = order.type === INCREASE ? t`Increase` : t`Decrease`
const longShortText = order.isLong ? t`Long` : t`Short`;

const orderTitle = `${orderTypeText} ${indexTokenSymbol} ${longShortText}  by ${formatAmount(order.sizeDelta, USD_DECIMALS, 2, true)}`

```